widgets/repo/40_attentionPRs.js

```diff
@@ -23,14 +23,9 @@ module.exports = function(options, callback) {
         })
         scores.sort((b,a) => a.score - b.score);
 
-        // remove PRs that are too new
-        if (options.config.hoursToRespond) {
```

Comment:
I agree that showing all PRs would be useful. If `hoursToRespond` is used, it may make sense to show those PRs, but delineate which have been open less that `hoursToRespond`. Thoughts?

Reply:
I think adding a divider would be a good idea, if there are PRs which are newer than `hoursToRespond` and PRs which are older.

Reply:
👍

Reply:
btw, hi @stuartlangridge ! :)

Reply:
:wave: ! Heya!

Reply:
Looking at the filter for hoursToRespond, shouldn't we be using the `substract` method of `moment`, rather than `add`? What I understand that happens now is that we `htr` is being assigned to a moment in the future, so we wouldn't be filtering anything at this point.

Reply:
There's a minus sign in front of hoursToRespond, you'll notice. Probably `.subtract(options.config.hoursToRespond, "hours")` would be cleaner. Not sure I knew that `.subtract` existed :)

Reply:
Thanks @stuartlangridge ! I had indeed missed the minus sign.